### DISCUSSION TITLE
Recipients of collected contacts Roundcube Webmail

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17273] [How to] [answered]
Hello,

Any way to collect email address when sent from my mailbox?

That is, in Roundcube Webmail when an email is sent to an address that is not in the contact list, it leaves that email address registered.

How to achieve this in yetiforce?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
is possible, requires changes to the code

**Reply by krseguros** [reply]
Thank you
